Sources/CodeEditTextView/Controller/STTextViewController.swift

```diff
@@ -214,11 +226,17 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
     // MARK: UI
 
     /// A default `NSParagraphStyle` with a set `lineHeight`
-    private var paragraphStyle: NSMutableParagraphStyle {
+    private lazy var paragraphStyle: NSMutableParagraphStyle = generateParagraphStyle()
+
+    private func generateParagraphStyle() -> NSMutableParagraphStyle {
         // swiftlint:disable:next force_cast
         let paragraph = NSParagraphStyle.default.mutableCopy() as! NSMutableParagraphStyle
         paragraph.minimumLineHeight = lineHeight
         paragraph.maximumLineHeight = lineHeight
+        // TODO: Fix Tab widths
```

Comment:
What is wrong here? Why is this a todo? I assume something is not working with the code below. What needs fixing? Please not this in the comment.

Reply:
This makes tabs work as they would in a normal document (eg there are tab stops every 0.25" but in this case every couple pixels). I'm leaving it as TODO while I look for a solution that works in case someone else has an idea. I can note what's wrong in a comment.

Reply:
Comment added below the TODO.